.github/workflows/cross_platform_ci.yml

```diff
@@ -0,0 +1,101 @@
+name: Build Moveit Core on all platforms
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel"
+
+jobs:
+  run_moveit_core_compilation:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Moveit Core on Unix
```

Comment:
Can we employ ccache here as well? See our [ci.yaml](https://github.com/ros-planning/moveit/blob/f62c30f61e539541c7a7d0387db75c559b0974e2/.github/workflows/ci.yaml#L75-L82).

Reply:
Unfortunately using `ccache` alongside `conda` is still an open issue / experimental; see https://github.com/conda-forge/conda-forge.github.io/issues/389 and https://github.com/conda-forge/compilers-feedstock/issues/31. I'd rather not include this for now, as it might cause unforeseen issues. What do you think @wolfv?

Reply:
yes, I agree. We do have a ccache package on conda-forge and I want to figure out how to make use of it but I would defer this for later. Also this CI pipeline is quite fast as it only builds moveit-core for now. Once we start adding the other moveit packages it would make sense to re-visit ccache.